www/data_service/src/md_demo/demo.controller.coffee

```diff
@@ -0,0 +1,9 @@
+class Demo extends Controller('app')
+    constructor: ($scope, $timeout, $log, dataService) ->
+        opened = dataService.open()
+        opened.getBuilders().then (@builders) =>
```

Comment:
would like to make it possible to do:

     opened.getBuilders(subscribe:false).then (@builders) =>

subscription is expensive and optionally disabled